src/Raven.Server/Utils/RavenCli.cs

```diff
@@ -537,19 +537,27 @@ private static bool CommandScript(List<string> args, RavenCli cli)
             {
                 object result;
                 var adminJsScript = new AdminJsScript { Script = jsCli.Script };
+                var certificate = cli._server.ServerCertificateHolder.Certificate.FriendlyName;
```

Comment:
No, this is bad, you are saying that the server itself run this, but you can just say that this is coming via the pipe